tutor/src/models/hypothesis.js

```diff
@@ -0,0 +1,62 @@
+import {
+  BaseModel, field, identifiedBy
+} from './base';
+import { action, observable, when, computed } from 'mobx';
+import loadjs from 'loadjs';
+import {Logging } from 'shared';
+
+@identifiedBy('hypotheis')
+class Hypothesis extends BaseModel {
+
+  @action.bound
+  bootstrap(data) {
+    this.update(data);
+  }
+
+  @field embed_url;
+  @field api_url;
+  @field authority;
+  @field grant_token;
+
+  logFailure(msg) {
+    this.errorMessage = msg;
+    this.isBusy = false;
+    Logging.error(msg)
+  }
+
```

Comment:
`errorMessage` and `isBusy` should be declared as an `@observable` so that components will re-render when they're updated.  Also `logFailure` should be a `@action` so that mobx only fires a singe update, rather than two as each variable is updated.